connector/src/test/scala/com/google/cloud/spark/bigquery/DirectBigQueryRelationSuite.scala

```diff
@@ -158,43 +164,52 @@ class DirectBigQueryRelationSuite
   }
 
   test("old filter behaviour, with filter option") {
-    val r = new DirectBigQueryRelation(
-      SparkBigQueryOptions(
-        ID, PROJECT_ID, combinePushedDownFilters = false, filter = Some("f>1")
-      ), TABLE)(sqlCtx)
+    val options = defaultOptions
+    options.combinePushedDownFilters = false
```

Comment:
should we have a test case for when these filters are combined?

Reply:
We have, line 181